Assets/_Project/Code/Models/Category.cs

```diff
@@ -0,0 +1,66 @@
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using System;
+
+public class Category
+{
+    [JsonProperty("nazwa")]
+    public string Nazwa { get; }
+    [JsonProperty("pytania", Order = 2)]
+    public List<Question> questionList;
+
+    public Category(string nazwa)
+    {
+        Nazwa = nazwa;
+        questionList = new List<Question>();
+    }
+
+    [JsonConstructor]
+    public Category(string nazwa, List<Question> lista)
+    {
+        this.Nazwa = nazwa;
+        this.questionList = lista;
+    }
+
+    public void DodajPytanieDoListy(Question pytanie)
+    {
+        if (pytanie == null)
+        {
+            return;
+        }
+        questionList.Add(pytanie);
+    }
+
+    public Question LosujPytanie()
+    {
+        System.Random random = new System.Random();
```

Comment:
tworzysz nowy obiekt random gdy wywołujesz LosujPytanie, zrób to jako pole i nawet mozesz dać statyczne żeby było używane pomiędzy instancjami